.github/workflows/repair-pr-805-runtime-accounting.yml

```
@@ -0,0 +1,602 @@
+        run: python -m pip install --require-hashes -r fuzz/requirements-property.txt
+
+      - name: Add regressions first and prove RED
+        shell: bash
+        run: |
+          python - <<'PY'
+          from pathlib import Path
+
+          judge_path = Path("tests/test_model_judge.py")
+          judge = judge_path.read_text(encoding="utf-8")
+          old_judge_test = '''def test_fast_mlsirm_adapter_routes_structured_completion_through_gateway() -> None:
+    orchestrator, _ = _orch("unused")
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)